libs/sdk-ui-dashboard/src/model/store/alerts/alertsSelectors.ts

```diff
@@ -35,32 +45,54 @@ interface IWidgetAlertMapItem {
  *
  * @internal
  */
-const selectAlertsMapByWidgetRefs = createSelector(selectAlerts, selectWidgetsMap, (alerts, widgetsMap) => {
-    const mappedItems = compact(
-        alerts.map((alert) => {
-            const widget = widgetsMap.get(alert.widget);
-            if (!widget) {
-                /**
-                 * Ignore widgets that are no longer available, this can naturally happen in edit mode when
-                 * a KPI widget is deleted by the user: the alerts are not removed from redux in case the edit
-                 * mode is cancelled (to avoid having to load the dashboard again from the backend).
-                 */
-                return undefined;
-            }
+const selectAlertsMapByWidgetRefs = createSelector(
+    selectAlerts,
```

Comment:
nOt sure about this fix SDK only support tiger so no alerts ... if this fix necessary for bear than we need to propagate to different branch